### PR TITLE
Fix authentication according to review

### PR DESCRIPTION
**Pull request by davidDoskalovich07**

### CODE REVIEW

**Review by N4tty93 — APPROVED** [review]

File: apps/kol-amelamdim/pages/login/index.tsx [hunk 1]
```diff
@@ -70,6 +77,10 @@ const Login = () => {
             </Button>
           </Grid>
         </form>
+        <Typography component="h4">
+          עוד אין לכם משתמש ?{' '}
+          <Link onClick={() => router.push('/register')}>לחצו עלי להרשמה</Link>
+        </Typography>
```
Comment (N4tty93) [inline apps/kol-amelamdim/pages/login/index.tsx]: The Link should wrap the Typography. That way we can show the user it's clickable. right now It's shown with normal cursor and not pointer.

File: apps/kol-amelamdim/pages/register/index.tsx [hunk 2]
```diff
@@ -69,6 +76,12 @@ const Register = () => {
             </Button>
           </Grid>
         </form>
+        <Typography component="h4">
+          כבר נרשמתם ?{' '}
+          <Link onClick={() => router.push('/register')}>
+            לחצו עלי להתחברות
+          </Link>
+        </Typography>
```
Comment (N4tty93) [inline apps/kol-amelamdim/pages/register/index.tsx]: Same here we should wrap the Typography with Link

Comment (N4tty93) [reply apps/kol-amelamdim/pages/register/index.tsx]: also, you can add sx={{mt: 2}} or something to make it spaced out from the button

File: apps/kol-amelamdim/pages/login/index.tsx (hunk 1 above)
Comment (davidDoskalovich07) [reply apps/kol-amelamdim/pages/login/index.tsx]: Resolved.

File: apps/kol-amelamdim/pages/register/index.tsx (hunk 2 above)
Comment (davidDoskalovich07) [reply apps/kol-amelamdim/pages/register/index.tsx]: Resolved.

**Pull request merged**